### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sindhus**
Add instructions on how to run tests

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
...
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
+
+Ensure you have PySQLite installed before you run:
+
+   $ coverage run `which nosetests`
+
+To run a single test:
+
+   $ nosetests tests/<test_filename.py> # Example: `nosetests test/test_render_with.py`
```
Comment (jace) [inline README.rst]: This should be `nosetests tests.test_render_with`. It expects a module name.

File: README.rst [hunk 2]
```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
...
+
+### Run tests
+
+You can also look at `.travis.yml` for instructions on how to run tests.
+Create a test Coaster DB:
+
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
+
+Ensure you have PySQLite installed before you run:
```
Comment (jace) [inline README.rst]: Remove this and add above: `pip install -r test_requirements.txt`

File: README.rst [hunk 3]
```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
 
 Coaster contains functions and db models for recurring patterns in Flask
 apps. Documentation at http://coaster.readthedocs.org/
+
+### Run tests
+
+You can also look at `.travis.yml` for instructions on how to run tests.
```
Comment (jace) [inline README.rst]: Remove this line. We don't want to confuse people with Travis-specific instructions.

File: README.rst [hunk 4]
```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
...
+Create a test Coaster DB:
+
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
+
+Ensure you have PySQLite installed before you run:
+
+   $ coverage run `which nosetests`
```
Comment (jace) [inline README.rst]: Just `nosetests` is sufficient. Coverage is for getting code-coverage reports, an advanced use case for someone looking for code that isn't yet tested.

File: tests/settings.py [hunk 5]
```diff
@@ -9,4 +9,4 @@
 MAIL_USERNAME = 'username'
 MAIL_PASSWORD = 'PASSWORD'
 SECRET_KEY = 'd vldvnvnvjn'
-SQLALCHEMY_DATABASE_URI = 'postgresql://:@localhost:5432/coaster_test'
+SQLALCHEMY_DATABASE_URI = 'postgresql://coaster:@localhost:5432/coaster_test'
```
Comment (jace) [inline tests/settings.py]: This change is not required as the file has dummy data.

File: README.rst [hunk 6]
```diff
@@ -10,3 +10,22 @@ Coaster: common patterns for Flask apps
 
 Coaster contains functions and db models for recurring patterns in Flask
 apps. Documentation at http://coaster.readthedocs.org/
+
+### Run tests
+
+Create a test Coaster DB:
+
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
```
Comment (jace) [inline README.rst]: I just confirmed that `tests/test_models.py` does not specify `coaster` as the username in the database connection. It defaults to the currently logged in user (see `whoami`) and Travis sets this as well, so the instructions should be amended accordingly.

```shell
$ createuser `whoami`
$ createdb -O `whoami` coaster_test
```

Comment (sindhus) [inline README.rst]: Done.

**Pull request merged**